### ISSUE TITLE
Ansible does not work with Vagrant setup on Windows

### ISSUE THREAD

**Issue by dubeejw** [openwhisk/openwhisk#601]
Ansible fails when running through a Vagrant setup on Windows. It appears to that Ansible does not work properly with shared folders in a VM running on top of Windows. See the following Ansible issue for more details: https://github.com/ansible/ansible/issues/9526.

Error output: 
fatal: [ansible]: FAILED! => {"changed": true, "failed": true, "msg": "Could not replace file: /home/vagrant/.ansible/tmp/ansible-tmp-1465332393.44-5014655483320/source to /home/vagrant/openwhisk/ansible/../whisk.properties: [Errno 26]

**Comment by domdom82** [comment]
@dubeejw we don't have windows machines here to test this, however can you try the proposed solution by setting the remote_tmp folder to something outside /vagrant which is not mapped in the shared folder? I am guessing /tmp could work.

**Comment by domdom82** [comment]
@dubeejw any updates on this?

**Comment by csantanapr** [comment]
@domdom82 We are waiting on you 😄 
Please provide things to try (what files to change) to see if it will fix it

**Comment by domdom82** [comment]
Proposed solution: Edit ansible.cfg in your openwhisk/ansible dir as follows:

```
[defaults]
(...)
remote_tmp = /tmp/ansible/remote_tmp
local_tmp = /tmp/ansible/local_tmp
```

**Comment by csantanapr** [comment]
Thanks @domdom82 for the info, @dubeejw give it a try and report back.

**Comment by domdom82** [comment]
final solution is to export ANSIBLE_REMOTE_TEMP=/home/vagrant/openwhisk/ansible/tmp for vagrant installations. fixed in #605
